packages/main/src/Select.hbs

```diff
@@ -9,7 +9,7 @@
 		<ui5-popover
 			placement-type="Bottom"
 			hide-header="true"
-			hide-arrow="true"
+			arrow-hiden="true"
```

Comment:
There are still "arrow-hidden" occurrences left - here and below

Reply:
can you please fix these?